index.d.ts

```diff
@@ -0,0 +1,1987 @@
+        test: boolean;
+        user_id: number;
+        currency: string;
+    }
+
+    interface IUsageCharge {
+        balance_remaining: number;
+        balance_used: number;
+        billing_on: string;
+        created_at: string;
+        description: string;
+        id: number;
+        price: string;
+        risk_level: number
+        recurring_application_charge_id: number;
+        updated_at: string;
+    }
+
+    interface ICreateUsageCharge {
+        description: string;
+        price: number;
+    }
+
+    type UserPermissions =
+        "applications" |
+        "customers" |
+        "dashboard" |
+        "full" |
+        "gift_cards" |
+        "links" |
+        "marketing" |
+        "order" |
+        "pages" |
+        "preferences" |
+        "products" |
+        "reports" |
+        "themes";
+
+    type UserType = "regular" | "restricted";
+
+    interface IUser {
+        account_owner: boolean;
+        bio: string;
+        email: string;
+        first_name: string;
+        id: number;
+        im: string;
+        last_name: string;
+        permissions: UserPermissions[];
+        phone: string;
+        pin: string;
+        receive_announcements: number;
+        screen_name: string;
+        url: string;
+        user_type: UserType;
+    }
+
+    export type WebhookTopic =
+        "app/uninstalled" |
+        "carts/create" |
+        "carts/update" |
+        "checkouts/create" |
+        "checkouts/delete" |
+        "checkouts/update" |
+        "collection_listings/add" |
+        "collection_listings/remove" |
+        "collection_listings/update" |
+        "collections/create" |
+        "collections/delete" |
+        "collections/update" |
+        "customer_groups/create" |
+        "customer_groups/delete" |
+        "customer_groups/update" |
+        "customers/create" |
+        "customers/delete" |
+        "customers/disable" |
+        "customers/enable" |
+        "customers/update" |
+        "draft_orders/create" |
+        "draft_orders/delete" |
+        "draft_orders/update" |
+        "fulfillment_events/create" |
+        "fulfillment_events/delete" |
+        "fulfillments/create" |
+        "fulfillments/update" |
+        "order_transactions/create" |
+        "orders/cancelled" |
+        "orders/create" |
+        "orders/delete" |
+        "orders/fulfilled" |
+        "orders/paid" |
+        "orders/partially_fulfilled" |
+        "orders/updated" |
+        "product_listings/add" |
+        "product_listings/remove" |
+        "product_listings/update" |
+        "products/create" |
+        "products/delete" |
+        "products/update" |
+        "refunds/create" |
+        "shop/update" |
+        "themes/create" |
+        "themes/delete" |
+        "themes/publish" |
+        "themes/update";
+
+    type WebhookFormat = "json" | "xml";
+
+    export interface IWebhook {
+        address: string;
+        created_at: string;
+        fields: s
```

Comment:
Please add a new line.

Reply:
Not sure why but this comment seems misplaced, I meant new line at the end of file.